### PR TITLE
next/327/20240304/v1

### PR DESCRIPTION
**Pull request by victorjulien**
Staging:
 - #10562
 - #10567

SV_BRANCH=https://github.com/OISF/suricata-verify/pull/1680

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information:
 

ERROR: QA failed on SURI_TLPW2_autofp_suri_time.

ERROR: QA failed on SURI_TLPR1_suri_time.
 | field | baseline | test | % |
|--|--|--|--| 
| | SURI_TLPW2_autofp_stats_chk | | |
| .uptime | 137 | 143 | 104.38% |
| | SURI_TLPR1_stats_chk | | |
| .uptime | 754 | 794 | 105.31% | 

Pipeline 18958

**Comment by victorjulien** [comment]
@ct0br0 can this get a rerun?

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 18972

File: src/interval-tree.h [hunk 1]
```diff
@@ -0,0 +1,563 @@
+/*	$NetBSD: interval-tree.h,v 1.8 2004/03/28 19:38:30 provos Exp $	*/
+/*	$OpenBSD: interval-tree.h,v 1.7 2002/10/17 21:51:54 art Exp $	*/
+/* $FreeBSD$ */
+
+/* This is a COPY of the in-tree tree.h modified to accomodate interval
+ * tree operations */
```
Comment (jasonish) [inline src/interval-tree.h:6]: I guess I would like the location and revision of the original file as well? Could be another PR though.

**Review by jasonish — APPROVED** [review]
